bundles/org.openhab.binding.tesla/src/main/java/org/openhab/binding/tesla/internal/protocol/dto/VehicleData.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.tesla.internal.protocol.dto;
+
+import java.util.List;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link VehicleData} is a data structure to capture
+ * variables sent by the Tesla API about a vehicle.
+ *
+ * @author Kai Kreuzer - Initial contribution
+ */
+public class VehicleData {
+
+    @SerializedName("id")
+    public Long id;
+    @SerializedName("user_id")
+    public int userId;
+    @SerializedName("vehicle_id")
+    public int vehicleId;
```

Comment:
This used to be a String and `int` is causing parsing exceptions. I assume this was changed by mistake?
```suggestion
    public String vehicleId;
```